scripts/inline_threads.py

```
@@ -0,0 +1,546 @@
+    def normalize(s):
+        return set(re.findall(r"[\w$]+", s.lower()))
+
+    results = {}
+    for b in bullets:
+        path, ln, le = b["path"], b["line"], b["line_end"]
+        lines = anchors.get(path, set())
+        text = normalize(b["body"])
+        if not text:
+            continue
+        best, best_score = None, 0.0
+        for l in lines:
```

Ruff (0.16.2)

[error] Ambiguous variable name: `l`

(E741)